delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutes.scala

```diff
@@ -0,0 +1,206 @@
+import ch.epfl.bluebrain.nexus.delta.routes.models.{JsonSource, Tag, Tags}
+import ch.epfl.bluebrain.nexus.delta.sdk.Permissions.{events, resolvers => resolverPermissions}
+import ch.epfl.bluebrain.nexus.delta.sdk._
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.AuthDirectives
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclAddress
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Caller
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.PaginationConfig
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchParams.ResolverSearchParams
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults.{searchResultsEncoder, SearchEncoder}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, IdSegment, Label}
+import io.circe.Json
+import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import monix.execution.Scheduler
+
+/**
+  * The resolver routes
+  * @param identities the identity module
+  * @param acls  the acls module
+  * @param projects the projects module
+  * @param resolvers the resolvers module
+  */
+final class ResolversRoutes(identities: Identities, acls: Acls, projects: Projects, resolvers: Resolvers)(implicit
+    baseUri: BaseUri,
+    paginationConfig: PaginationConfig,
+    s: Scheduler,
+    cr: RemoteContextResolution,
+    ordering: JsonKeyOrdering
+) extends AuthDirectives(identities, acls)
+    with CirceUnmarshalling {
+
+  import baseUri.prefixSegment
+  implicit val resolverContext: ContextValue = Resolvers.context
+
+  private val Read  = resolverPermissions.read
+  private val Write = resolverPermissions.write
+
+  private def resolverSearchParams(project: ProjectRef)(implicit caller: Caller): Directive1[ResolverSearchParams] =
+    searchParams.tflatMap { case (deprecated, rev, createdBy, updatedBy) =>
+      callerAcls.map { aclsCol =>
+        ResolverSearchParams(
+          Some(project),
+          deprecated,
+          rev,
+          createdBy,
+          updatedBy,
+          resolver => aclsCol.exists(caller.identities, Read, AclAddress.Project(resolver.project))
+        )
+      }
+    }
+
+  def routes: Route =
+    baseUriPrefix(baseUri.prefix) {
+      extractCaller { implicit caller =>
+        pathPrefix("resolvers") {
+          concat(
+            // SSE resolvers for all events
+            (pathPrefix("events") & pathEndOrSingleSlash) {
+              get {
+                operationName(s"$prefixSegment/resolvers/events") {
+                  authorizeFor(AclAddress.Root, events.read).apply {
+                    lastEventId { offset =>
+                      emit(resolvers.events(offset))
+                    }
+                  }
+                }
+              }
+            },
+            (projectRef | projectRefFromUuidsLookup(projects)) { ref =>
```

Comment:
Where are the SSEs on:
- /resolvers/{org}/events
- /resolvers/{org}/{proj}/events

Reply:
We keep it for 1.6.0 according to this: #997

Reply:
I'm fine with doing this later. However it is already done for resources and schemas. 

It seems quite trivial to do it for the rest of resources since we are currently building the routes. It is one of the situations where doing it now will be probably faster than later.

Reply:
I would like to start the most important part, resource resolution asap.
This is nice to have but we can go back to this later.